src/common/configuration.h

```diff
@@ -71,11 +71,7 @@ DECLARE_JSON_OPTIONAL_FIELDS(CCFConfig::JWT, key_refresh_interval);
 DECLARE_JSON_TYPE_WITH_OPTIONAL_FIELDS(CCFConfig::Attestation::Environment);
 DECLARE_JSON_REQUIRED_FIELDS(CCFConfig::Attestation::Environment);
 DECLARE_JSON_OPTIONAL_FIELDS(
-  CCFConfig::Attestation::Environment,
-  security_context_directory,
-  security_policy,
-  uvm_endorsements,
-  report_endorsements);
+  CCFConfig::Attestation::Environment, security_policy, uvm_endorsements);
```

Comment:
Environment remains as an internal implementation detail for, to carry the contents of the policy and the endorsements into the enclave. This can be removed once SGX support is dropped,